docs/UserGuide.md

```diff
@@ -202,7 +202,8 @@ that you have inputted. The values that you did not specify to edit will remain
 
 ### Billing a guest : `bill`
 
-This command updates the bill of a guest in GuestBook.
+This command updates the bill of a guest in GuestBook. Note that the bill here refers to the additional costs incurred
+by the guest during his stay at your hotel and should not include the cost of the hostel stay.
```

Comment:
Could we phrase this as "Note that the bill here refers to the total additional cost incurred by the guest during their stay at your hotel and should not include the cost of the room booking."

Reply:
Fixed!